code/controllers/subsystem/ships.dm

```diff
@@ -116,17 +116,21 @@ var/global/list/ftl_weapons_consoles = list()
 			return
 		if(S.planet != SSstarmap.current_planet)
 			return
-		if(world.time > S.next_attack && S.fire_rate)
-			S.next_attack = world.time + S.fire_rate
-			attack_player(S,pick(get_attacks(S)))
+		for(var/datum/component/weapon/W in S.components)
+			if(world.time > W.next_attack && W.fire_rate)
+				W.next_attack = world.time + W.fire_rate + rand(1,50)
```

Comment:
how much? I changed it to a 100, keep in mind that after each attack the difference will keep getting bigger.